content/waku/2024-gantt.md

```diff
@@ -0,0 +1,100 @@
+        (metric) Count store messages: milestone, after count-store-msg, 0
+    section Direct Message Reliability
+        (testing) direct messages: milestone, after test-direct-msg, 0
+        Review connection management: crit, milestone, after review-conn-mgmt-1 review-conn-mgmt-2, 0
+        (telemetry) Fleet logging: milestone, after telem-fleet-logging, 0
+        (telemetry) direct message reliability: milestone, after telem-d-msg-rel, 0
+        Reliability Protocol for Relay: crit, milestone, after rel-relay-g rel-relay-n, 0
+        Reliability Protocol for Resource-Restricted Clients: crit, milestone, after rel-reqres-g rel-reqres-j, 0
+        Review MVDS usage and fail path: crit, milestone, after mvds, 0
+    section Static Sharding - dedicated shards
+        (telemetry) Measure Bandwidth: milestone, after telem-bandwidth, 0
+        Sharding peer mgmt and discovery hardening: crit, milestone, after sh-peer-mgmt-j sh-peer-mgmt-n sh-peer-mgmt-g, 0
+        (testing) Custom shard impl of Communities: milestone, after test-custom-shard, 0
+
+%% Tasks
+    section golang eng 1
+        %% Estimation TBC - Prem says fine, waiting on 2nd opinion
+        TBC Review connection management: crit, review-conn-mgmt-1, 2024-05-13, 8w
+        Review MVDS usage and fail path: crit, mvds, after review-conn-mgmt-1, 6w
+        (testing) direct messages: test-direct-msg, after mvds, 4w
+        TBC (testing) Custom shard impl of Communities: test-custom-shard, after test-direct-msg, 4w 
+    section golang eng 2
+        Reliability Protocol for Relay (status-go): crit, rel-relay-g, 2024-05-13, 12w
+        Investigation and fixing of bugs discovered during dogfooding/usage/simulations: go-bugs-1, after rel-relay-g, 8w
+    section golang eng 3
+        Store v3 (go-waku client only): crit, storev3-g, 2024-02-26, 2024-05-24
+        %% Estimate TBC - assuming parallel work possible
+        TBC Review connection management: crit, review-conn-mgmt-2, after storev3-g, 8w
+        Sharding peer mgmt and discovery hardening (go-waku): crit, sh-peer-mgmt-g, after review-conn-mgmt-2, 8w
+    section golang eng 4
+        Store v3 (sync): crit, 2024-02-08, 2024-04-26
+        Reliability Protocol for Resource-Restricted Clients (status-go): crit, rel-reqres-g, 2024-05-13, 10w
+        (metric) Count store messages: count-store-msg, after rel-reqres-g, 2w
+        Investigation and fixing of bugs discovered during dogfooding/usage/simulations: go-bugs-2, after count-store-msg, 8w
+    section golang eng 5
+        DoS protection for req-res protocols (go-waku client only): crit, dosreqresg, 2024-05-20, 4w
+        (telemetry) direct message reliability: telem-d-msg-rel, 2024-06-17, 6w
+        (telemetry) Measure Bandwidth: telem-bandwidth, after telem-d-msg-rel, 8w
+    section test eng 1 (TBC)
+        %% TBC estimate
+        Peer and connection management tests: sim-conn-mgmt, 2024-05-13, 4w
```

Comment:
Shall we have 2 items? one sharded, one none sharded?

Reply:
I think that will be an unnecessary division - we're just looking at different scenarios with more or fewer shards down to a minimum of one, but with a shared block of management code.